### PR TITLE
feat(invite-dialog) Add limit message for invite dialog

### PR DESCRIPTION
**Pull request by apetrus20**

### CODE REVIEW

File: react/features/invite/components/add-people-dialog/web/DialInLimit.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            borderRadius: '6px',
+            padding: '8px 16px'
+        },
+        limitInfo: {
+            color: theme.palette.field01,
+            fontFamily: 'Inter',
+            fontStyle: 'normal',
+            fontWeight: 'normal',
+            fontSize: '14px',
+            lineHeight: '20px'
```
Comment (horymury) [inline react/features/invite/components/add-people-dialog/web/DialInLimit.tsx:22]: use theme token:  `theme.typography.bodyShortRegular` : check examples in code, e.g.: `...withPixelLineHeight(theme.typography.bodyShortRegular)`

File: react/features/invite/components/add-people-dialog/web/DialInLimit.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+const useStyles = makeStyles()((theme: Theme) => {
+    return {
+        limitContainer: {
+            backgroundColor: theme.palette.warning01,
+            borderRadius: '6px',
+            padding: '8px 16px'
+        },
+        limitInfo: {
+            color: theme.palette.field01,
+            fontFamily: 'Inter',
```
Comment (horymury) [inline react/features/invite/components/add-people-dialog/web/DialInLimit.tsx:18]: isn't the font inherited?

File: react/features/invite/components/add-people-dialog/web/DialInLimit.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+/**
+ * Component that displays a message when the dial in limit is reached.
+ * * @param {Function} t - Function which translate strings.
+ *
+ * @returns {ReactElement}
+ */
+const DialInLimit: React.FC<WithTranslation> = ({ t }) => {
+    const { classes } = useStyles();
+    const upgradeOptionsText = 'jaas.8x8.vc';
+    const upgradeOptionsLink = 'https://jaas.8x8.vc/#/plan/upgrade';
```
Comment (horymury) [inline react/features/invite/components/add-people-dialog/web/DialInLimit.tsx:41]: move constants definitions in specialized files  eg `constants.js`

File: react/features/invite/components/add-people-dialog/web/AddPeopleDialog.js [hunk 4]
```diff
@@ -183,6 +184,9 @@ function AddPeopleDialog({
                         && _dialInVisible
                         && <DialInSection phoneNumber = { _phoneNumber } />
                 }
+                {
+                    !_dialInVisible && <DialInLimit />
```
Comment (horymury) [inline react/features/invite/components/add-people-dialog/web/AddPeopleDialog.js:188]: do we want this dialog for meet.jit.si as well?

**Review by horymury — CHANGES_REQUESTED** [review]

File: lang/main.json [hunk 5]
```diff
@@ -477,9 +477,11 @@
         "noRoom": "No room was specified to dial-in into.",
         "numbers": "Dial-in Numbers",
         "password": "$t(lockRoomPasswordUppercase): ",
+        "reachedLimit": "you have reached the limit for you plan.",
         "sip": "SIP address",
         "title": "Share",
-        "tooltip": "Share link and dial-in info for this meeting"
+        "tooltip": "Share link and dial-in info for this meeting",
+        "upgradeOptions": "Please check the upgrade  options on"
```
Comment (saghul) [inline lang/main.json:484]: nit: extra space

**Review by horymury — APPROVED** [review]

**Pull request merged**